apps/web/src/forms/SymptomChecker.tsx

```diff
@@ -65,6 +65,11 @@ export const SymptomChecker = () => {
       return step + 1;
     }
 
+    // exit if no symptoms
+    if (values.symptoms.none.checked === true) {
+      return recommend(Recommendation.ASYMPTOMATIC_NO_TEST);
+    }
```

Comment:
AC: When user answers None of the above to the symptoms question (Do you have any of the following symptoms?) preceding the vulnerable groups questions, they should immediately be recommended that they do not need a COVID test.

✅